src/v3/src/components/Widget/index.tsx

```diff
@@ -249,6 +249,11 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
       return createForm();
     }
 
+    // clear the resend reminder time stamp in session storage if current transaction does not have resend option
+    if (SessionStorage.getResendTimestamp() && !idxTransaction.nextStep?.canResend) {
```

Comment:
Is it necessary to check `getResendTimestamp` here?

Reply:
actually its not necessary here since `removeResendTimestamp()` will do nothing if it doesn't exist anyway